File: 프로그래머스/행렬의덧셈.cpp
#include <string>
#include <vector>
#include <iostream>

using namespace std;

vector<vector<int>> solution(vector<vector<int>> arr1, vector<vector<int>> arr2) {
	vector<vector<int>> answer;

	for (int i = 0; i < arr1.size(); i++) {
		vector<int> v;

		for (int j = 0; j < arr1[0].size(); j++) {
			v.push_back(arr1[i][j] + arr2[i][j]);
		}
		answer.push_back(v);
	}
	return answer;
}

int main() {
	cout << solution({ {1, 2}, {2, 3} }, { {3, 4}, {5, 6} });
}

File: 프로그래머스/행렬의덧셈.py
def solution(arr1, arr2):
    answer = []
    v = []
    for i in range(len(arr1)):
        for j in range(len(arr1[i])):
            v.append(arr1[i][j]+arr2[i][j])
        answer.append(v)
        v = []
    return answer


ar1 = [[1, 2], [2, 3]]
ar2 = [[3, 4], [5, 6]]
print(solution(ar1, ar2))
